extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -105,7 +109,20 @@ class AmpStickyAd extends AMP.BaseElement {
         toggle(this.element, true);
         this.viewport_.addToFixedLayer(this.element);
         this.scheduleLayout(this.ad_);
+        // Add border-bottom to the body to compensate space that was taken
+        // by sticky ad, so no content would be blocked by sticky ad unit.
+        const borderBottom = this.element./*OK*/offsetHeight;
+        this.viewport_.updatePaddingBottom(borderBottom);
+        // TODO(zhouyx): need to delete borderBottom when sticky ad is dismissed
         this.removeOnScrollListener_();
+        timer.delay(() => {
+          // Unfortunately we don't really have a good way to measure how long it
+          // takes to load an ad, so we'll just pretend it takes 1 second for
+          // now.
+          this.vsync_.mutate(() => {
+            this.element.classList.add('amp-sticky-ad-load');
```

Comment:
So, what's the meaning of this class? That the ad is loaded? If so, let's call it "amp-sticky-ad-loaded". If not - please clarify.